packages/legend-application-data-cube/src/stores/model/LocalFileDataCubeSource.ts

```diff
@@ -42,26 +41,20 @@ export class LocalFileDataCubeSource extends DataCubeSource {
 }
 
 export class RawLocalFileQueryDataCubeSource {
-  model!: PlainObject<V1_PureModelContextData>;
-  runtime!: string;
-  db!: string;
-  schema!: string;
-  table!: string;
-  count!: number;
   fileName!: string;
   fileFormat!: LocalFileDataCubeSourceFormat;
+  dbReference!: string;
+  count!: number;
```

Comment:
let's discuss about where you use this `count`

Reply:
We don't use it anywhere as of now, not sure if this will be required in the future at all for analytics